### DISCUSSION TITLE
Will validTime always be on the hour?

### DISCUSSION THREAD

**Discussion by Nogginboink** [weather-gov/api#449] [General]
When calling the forecast grid data endpoint, it appears to me that the validTime for each layer is always at zero minutes and zero seconds. 

Does the API make any guarantees on that?  It sure would be convenient, but I don't want to rely on something that, well, I shouldn't really rely on.

**Comment by StephenClouse** [comment]
It _should_ always be on the hour.

There's always a chance that bad data slips past the integrity checks, but this is a safe assumption at present.
